src/modules/users/components/GasStation/GasStationHeader/GasStationHeader.css

```diff
@@ -33,3 +35,51 @@
   composes: button from '~styles/reset.css';
   margin-left: 5px;
 }
+
+@media screen and query700 {
+  .main {
+    border-bottom-width: 1px;
+  }
+
+  .closeButton {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 26px;
+    width: 26px;
+    border: 1px solid color-mod(var(--dark) alpha(0.65));
+    border-radius: 50%;
+  }
+
+  .closeButton i {
+    transform: scale(1.3);
+  }
+
+  .closeButton svg {
+    fill: color-mod(var(--dark) alpha(0.65));
+  }
+
+  .goToWalletIcon {
+    display: none;
+  }
+
+  .walletAddress span:first-child {
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  .walletAddress span:nth-child(2) {
+    display: none;
+  }
```

Comment:
@rdig Is this what we're going for with the class selectors? If so I will use same pattern with other PRs and in general going forward.

Reply:
This is not ideal either, but definitely better than using hardcoded class names